docs/src/1_Mechanics/01_PlayerGuide_Full.md

```diff
@@ -181,10 +181,9 @@ the circles around the bullseye.
 
 Once the DR has been announced, players must make the check. Players may consult with
 their GM to determine if another Attribute or Skill may be used instead. A player will
-draw one or more cards. Immediately after drawing, a player will apply any relevant
+draw one card from the top of their deck. Immediately after drawing, a player will apply any relevant
```

Comment:
While as it stands people aren't drawing from anywhere but the top of their deck. But perhaps an effect in the future will let them do so, making the additional words important.

Reply:
Reverted the change. I think partly my reasoning here for modifying this is that since this is the first description of making a check in the PGF it may be confusing to a player wondering whether they draw a card, a few cards, whether there's a hand, etc. so I was thinking of finding a way to be very explicit up front that there's a deck of cards and you're top-decking to make a check.